src/diff/defaultRules.test.ts

```diff
@@ -27,7 +31,7 @@ function verifyRule(nodeChanges: NodeChanges, rule: Rule): void {
       nodeChanges.added[rule.event.params.changedProperty]
     ];
   }
-  expect(categorizedChanges).to.have.length(1);
+  expect(categorizedChanges).to.have.length(countOfChanges);
```

Comment:
length seems to have deprecated. We should use lengthOf instead

Reply:
done